src/Jackett.Common/Indexers/FunFile.cs

```diff
@@ -82,7 +82,7 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
             {
                 {"incldead", "1"},
                 {"showspam", "1"},
-                {"cat", MapTorznabCapsToTrackers(query).FirstOrDefault() ?? "0"}
+                {"cat", MapTorznabCapsToTrackers(query).DefaultIfEmpty("0").First()}
```

Comment:
This only gives us proper operation for 0 or 1 item in the list. If >=2 it still gives us only the first category which is the bug. There's no inline way to fix this, we need to pull `MapTorznabCapsToTrackers(query)` out to a variable, then use:
`cats.Count == 1 ? cats[0] : "0"`

Reply:
You are right, changed